package.json

```diff
@@ -83,5 +82,15 @@
   "bugs": {
     "url": ""
   },
-  "homepage": ""
+  "homepage": "",
+  "browserslist": "> 0.25%, not dead",
```

Comment:
Maybe it's good to match the browserslist query from our cli-style repo here https://github.com/dhis2/cli-style/blob/master/.browserslistrc

Reply:
I'll copy and add the browserslistrc in the root